// eth-chains/dist/src/enums.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ChainId = exports.ChainName = void 0;
var ChainName;
(function (ChainName) {
    ChainName["EthereumMainnet"] = "Ethereum Mainnet";
    ChainName["ExpanseNetwork"] = "Expanse Network";
    ChainName["Ropsten"] = "Ropsten";
    ChainName["Rinkeby"] = "Rinkeby";
    ChainName["Goerli"] = "G\u00F6rli";
    ChainName["EthereumClassicTestnetKotti"] = "Ethereum Classic Testnet Kotti";
    ChainName["ThaiChain"] = "ThaiChain";
    ChainName["Ubiq"] = "Ubiq";
    ChainName["UbiqNetworkTestnet"] = "Ubiq Network Testnet";
    ChainName["Optimism"] = "Optimism";
    ChainName["MetadiumMainnet"] = "Metadium Mainnet";
    ChainName["MetadiumTestnet"] = "Metadium Testnet";
    ChainName["DiodeTestnetStaging"] = "Diode Testnet Staging";
    ChainName["FlareMainnet"] = "Flare Mainnet";
    ChainName["DiodePrenet"] = "Diode Prenet";
    ChainName["FlareTestnetCoston"] = "Flare Testnet Coston";
    ChainName["ThaiChain2.0ThaiFi"] = "ThaiChain 2.0 ThaiFi";
    ChainName["ThunderCoreTestnet"] = "ThunderCore Testnet";
    ChainName["SongbirdCanary-Network"] = "Songbird Canary-Network";
    ChainName["ElastosSmartChain"] = "Elastos Smart Chain";
    ChainName["ELA-ETH-SidechainTestnet"] = "ELA-ETH-Sidechain Testnet";
    ChainName["ELA-DID-SidechainMainnet"] = "ELA-DID-Sidechain Mainnet";
    ChainName["ELA-DID-SidechainTestnet"] = "ELA-DID-Sidechain Testnet";
    ChainName["DithereumMainnet"] = "Dithereum Mainnet";
    ChainName["CronosMainnetBeta"] = "Cronos Mainnet Beta";
    ChainName["GenesisL1testnet"] = "Genesis L1 testnet";
    ChainName["ShibaChain"] = "ShibaChain";
    ChainName["BobaNetworkRinkebyTestnet"] = "Boba Network Rinkeby Testnet";
    ChainName["GenesisL1"] = "Genesis L1";
    ChainName["RSKMainnet"] = "RSK Mainnet";
    ChainName["RSKTestnet"] = "RSK Testnet";
    ChainName["GoodDataTestnet"] = "GoodData Testnet";
    ChainName["GoodDataMainnet"] = "GoodData Mainnet";
    ChainName["DithereumTestnet"] = "Dithereum Testnet";
    ChainName["TBWGChain"] = "TBWG Chain";
    ChainName["Valorbit"] = "Valorbit";
    ChainName["TelosEVMMainnet"] = "Telos EVM Mainnet";
    ChainName["TelosEVMTestnet"] = "Telos EVM Testnet";
    ChainName["Kovan"] = "Kovan";
    ChainName["DarwiniaPangolinTestnet"] = "Darwinia Pangolin Testnet";
    ChainName["DarwiniaCrabNetwork"] = "Darwinia Crab Network";
    ChainName["DarwiniaPangoroTestnet"] = "Darwinia Pangoro Testnet";
    ChainName["XinFinNetworkMainnet"] = "XinFin Network Mainnet";
    ChainName["XinFinApothemTestnet"] = "XinFin Apothem Testnet";
    ChainName["CoinExSmartChainMainnet"] = "CoinEx Smart Chain Mainnet";
    ChainName["CoinExSmartChainTestnet"] = "CoinEx Smart Chain Testnet";
    ChainName["OpenpieceMainnet"] = "Openpiece Mainnet";
    ChainName["ZyxMainnet"] = "Zyx Mainnet";
    ChainName["BinanceSmartChainMainnet"] = "Binance Smart Chain Mainnet";
    ChainName["SyscoinMainnet"] = "Syscoin Mainnet";
    ChainName["OntologyMainnet"] = "Ontology Mainnet";
    ChainName["EOSMainnet"] = "EOS Mainnet";
    ChainName["GoChain"] = "GoChain";
    ChainName["EthereumClassicMainnet"] = "Ethereum Classic Mainnet";
    ChainName["EthereumClassicTestnetMorden"] = "Ethereum Classic Testnet Morden";
    ChainName["EthereumClassicTestnetMordor"] = "Ethereum Classic Testnet Mordor";
    ChainName["Ellaism"] = "Ellaism";
    ChainName["OKExChainTestnet"] = "OKExChain Testnet";
    ChainName["OKXChainMainnet"] = "OKXChain Mainnet";
    ChainName["DBChainTestnet"] = "DBChain Testnet";
    ChainName["SoterOneMainnet"] = "SoterOne Mainnet";
    ChainName["OptimismKovan"] = "Optimism Kovan";
    ChainName["HooSmartChain"] = "Hoo Smart Chain";
    ChainName["ConfluxeSpace(Testnet)"] = "Conflux eSpace (Testnet)";
    ChainName["IDChainMainnet"] = "IDChain Mainnet";
    ChainName["Mix"] = "Mix";
    ChainName["POANetworkSokol"] = "POA Network Sokol";
    ChainName["PrimusChainmainnet"] = "PrimusChain mainnet";
    ChainName["ZenithMainnet"] = "Zenith Mainnet";
    ChainName["GeneChain"] = "GeneChain";
    ChainName["ZenithTestnet(Vilnius)"] = "Zenith Testnet (Vilnius)";
    ChainName["MeterMainnet"] = "Meter Mainnet";
    ChainName["MeterTestnet"] = "Meter Testnet";
    ChainName["GateChainTestnet"] = "GateChain Testnet";
    ChainName["GateChainMainnet"] = "GateChain Mainnet";
    ChainName["NovaNetwork"] = "Nova Network";
    ChainName["TomoChain"] = "TomoChain";
    ChainName["TomoChainTestnet"] = "TomoChain Testnet";
    ChainName["GarizonStage0"] = "Garizon Stage0";
    ChainName["GarizonStage1"] = "Garizon Stage1";
    ChainName["GarizonStage2"] = "Garizon Stage2";
    ChainName["GarizonStage3"] = "Garizon Stage3";
    ChainName["CryptoKylinTestnet"] = "CryptoKylin Testnet";
    ChainName["NEXTSmartChain"] = "NEXT Smart Chain";
    ChainName["BinanceSmartChainTestnet"] = "Binance Smart Chain Testnet";
    ChainName["POANetworkCore"] = "POA Network Core";
    ChainName["Gnosis"] = "Gnosis";
    ChainName["EtherInc"] = "EtherInc";
    ChainName["Web3GamesTestnet"] = "Web3Games Testnet";
    ChainName["Web3GamesDevnet"] = "Web3Games Devnet";
    ChainName["VelasEVMMainnet"] = "Velas EVM Mainnet";
    ChainName["NebulaTestnet"] = "Nebula Testnet";
    ChainName["ThunderCoreMainnet"] = "ThunderCore Mainnet";
    ChainName["ProtonTestnet"] = "Proton Testnet";
    ChainName["EtherLiteChain"] = "EtherLite Chain";
    ChainName["FuseMainnet"] = "Fuse Mainnet";
    ChainName["FuseSparknet"] = "Fuse Sparknet";
    ChainName["DecentralizedWebMainnet"] = "Decentralized Web Mainnet";
    ChainName["OYchainTestnet"] = "OYchain Testnet";
    ChainName["OYchainMainnet"] = "OYchain Mainnet";
    ChainName["Factory127Mainnet"] = "Factory 127 Mainnet";
    ChainName["HuobiECOChainMainnet"] = "Huobi ECO Chain Mainnet";
    ChainName["PolygonMainnet"] = "Polygon Mainnet";
    ChainName["OpenpieceTestnet"] = "Openpiece Testnet";
    ChainName["DAXCHAIN"] = "DAX CHAIN";
    ChainName["LightstreamsTestnet"] = "Lightstreams Testnet";
    ChainName["LightstreamsMainnet"] = "Lightstreams Mainnet";
    ChainName["AIOZNetwork"] = "AIOZ Network";
    ChainName["HOOSmartChainTestnet"] = "HOO Smart Chain Testnet";
    ChainName["Latam-BlockchainResilTestnet"] = "Latam-Blockchain Resil Testnet";
    ChainName["AMEChainMainnet"] = "AME Chain Mainnet";
    ChainName["SeeleMainnet"] = "Seele Mainnet";
    ChainName["BMCMainnet"] = "BMC Mainnet";
    ChainName["BMCTestnet"] = "BMC Testnet";
    ChainName["CryptoEmergency"] = "Crypto Emergency";
    ChainName["BitTorrentChainMainnet"] = "BitTorrent Chain Mainnet";
    ChainName["ArbitrumonxDai"] = "Arbitrum on xDai";
    ChainName["FreightTrustNetwork"] = "Freight Trust Network";
    ChainName["SoterOneMainnetold"] = "SoterOne Mainnet old";
    ChainName["Permission"] = "Permission";
    ChainName["LACHAINMainnet"] = "LACHAIN Mainnet";
    ChainName["LACHAINTestnet"] = "LACHAIN Testnet";
    ChainName["AitdTestnet"] = "Aitd Testnet";
    ChainName["EnergyWebChain"] = "Energy Web Chain";
    ChainName["FantomOpera"] = "Fantom Opera";
    ChainName["HuobiECOChainTestnet"] = "Huobi ECO Chain Testnet";
    ChainName["Setheum"] = "Setheum";
    ChainName["SURBlockchainNetwork"] = "SUR Blockchain Network";
    ChainName["HighPerformanceBlockchain"] = "High Performance Blockchain";
    ChainName["BobaNetwork"] = "Boba Network";
    ChainName["OptimismonGnosisChain"] = "Optimism on Gnosis Chain";
    ChainName["KCCMainnet"] = "KCC Mainnet";
    ChainName["KCCTestnet"] = "KCC Testnet";
    ChainName["Web3QMainnet"] = "Web3Q Mainnet";
    ChainName["DFKChainTest"] = "DFK Chain Test";
    ChainName["Shiden"] = "Shiden";
    ChainName["CronosTestnet"] = "Cronos Testnet";
    ChainName["ThetaMainnet"] = "Theta Mainnet";
    ChainName["ThetaSapphireTestnet"] = "Theta Sapphire Testnet";
    ChainName["ThetaAmberTestnet"] = "Theta Amber Testnet";
    ChainName["ThetaTestnet"] = "Theta Testnet";
    ChainName["PulseChainMainnet"] = "PulseChain Mainnet";
    ChainName["Lisinski"] = "Lisinski";
    ChainName["SXNetworkMainnet"] = "SX Network Mainnet";
    ChainName["OptimisticEthereumTestnetGoerli"] = "Optimistic Ethereum Testnet Goerli";
    ChainName["Rupaya"] = "Rupaya";
    ChainName["Double-AChainMainnet"] = "Double-A Chain Mainnet";
    ChainName["Double-AChainTestnet"] = "Double-A Chain Testnet";
    ChainName["XTSmartChainMainnet"] = "XT Smart Chain Mainnet";
    ChainName["Candle"] = "Candle";
    ChainName["Vela1ChainMainnet"] = "Vela1 Chain Mainnet";
    ChainName["TaoNetwork"] = "Tao Network";
    ChainName["MetisStardustTestnet"] = "Metis Stardust Testnet";
    ChainName["Astar"] = "Astar";
    ChainName["AcalaMandalaTestnet"] = "Acala Mandala Testnet";
    ChainName["KaruraNetworkTestnet"] = "Karura Network Testnet";
    ChainName["AcalaNetworkTestnet"] = "Acala Network Testnet";
    ChainName["Meshnyantestnet"] = "Meshnyan testnet";
    ChainName["SXNetworkTestnet"] = "SX Network Testnet";
    ChainName["PixieChainTestnet"] = "Pixie Chain Testnet";
    ChainName["KaruraNetwork"] = "Karura Network";
    ChainName["StarSocialTestnet"] = "Star Social Testnet";
    ChainName["BlockChainStationMainnet"] = "BlockChain Station Mainnet";
    ChainName["BlockChainStationTestnet"] = "BlockChain Station Testnet";
    ChainName["Factory127Testnet"] = "Factory 127 Testnet";
    ChainName["OpenChainTestnet"] = "OpenChain Testnet";
    ChainName["CheapETH"] = "cheapETH";
    ChainName["AcalaNetwork"] = "Acala Network";
    ChainName["AerochainTestnet"] = "Aerochain Testnet";
    ChainName["Haic"] = "Haic";
    ChainName["CallistoMainnet"] = "Callisto Mainnet";
    ChainName["CallistoTestnet"] = "Callisto Testnet";
    ChainName["AmbrosChainMainnet"] = "Ambros Chain Mainnet";
    ChainName["Wanchain"] = "Wanchain";
    ChainName["GarizonTestnetStage0"] = "Garizon Testnet Stage0";
    ChainName["GarizonTestnetStage1"] = "Garizon Testnet Stage1";
    ChainName["GarizonTestnetStage2"] = "Garizon Testnet Stage2";
    ChainName["GarizonTestnetStage3"] = "Garizon Testnet Stage3";
    ChainName["PulseChainTestnet"] = "PulseChain Testnet";
    ChainName["PulseChainTestnetv2b"] = "PulseChain Testnet v2b";
    ChainName["PulseChainTestnetv3"] = "PulseChain Testnet v3";
    ChainName["NepalBlockchainNetwork"] = "Nepal Blockchain Network";
    ChainName["TOPMainnetEVM"] = "TOP Mainnet EVM";
    ChainName["TOPMainnet"] = "TOP Mainnet";
    ChainName["LuckyNetwork"] = "Lucky Network";
    ChainName["WanchainTestnet"] = "Wanchain Testnet";
    ChainName["KlaytnTestnetBaobab"] = "Klaytn Testnet Baobab";
    ChainName["NewtonTestnet"] = "Newton Testnet";
    ChainName["EurusMainnet"] = "Eurus Mainnet";
    ChainName["EvriceNetwork"] = "Evrice Network";
    ChainName["Newton"] = "Newton";
    ChainName["Sakura"] = "Sakura";
    ChainName["CloverTestnet"] = "Clover Testnet";
    ChainName["CLVParachain"] = "CLV Parachain";
    ChainName["BitTorrentChainTestnet"] = "BitTorrent Chain Testnet";
    ChainName["ConfluxeSpace"] = "Conflux eSpace";
    ChainName["MetisAndromedaMainnet"] = "Metis Andromeda Mainnet";
    ChainName["MathChain"] = "MathChain";
    ChainName["MathChainTestnet"] = "MathChain Testnet";
    ChainName["IoraChain"] = "Iora Chain";
    ChainName["EvanescoTestnet"] = "Evanesco Testnet";
    ChainName["WorldTradeTechnicalChainMainnet"] = "World Trade Technical Chain Mainnet";
    ChainName["PopcateumMainnet"] = "Popcateum Mainnet";
    ChainName["EnterChainMainnet"] = "EnterChain Mainnet";
    ChainName["HALOMainnet"] = "HALO Mainnet";
    ChainName["Moonbeam"] = "Moonbeam";
    ChainName["Moonriver"] = "Moonriver";
    ChainName["Moonrockold"] = "Moonrock old";
    ChainName["MoonbaseAlpha"] = "Moonbase Alpha";
    ChainName["Moonrock"] = "Moonrock";
    ChainName["CENNZnetold"] = "CENNZnet old";
    ChainName["SherpaxMainnet"] = "Sherpax Mainnet";
    ChainName["SherpaxTestnet"] = "Sherpax Testnet";
    ChainName["CatecoinChainMainnet"] = "Catecoin Chain Mainnet";
    ChainName["Atheios"] = "Atheios";
    ChainName["Btachain"] = "Btachain";
    ChainName["LUDANMainnet"] = "LUDAN Mainnet";
    ChainName["CubeChainMainnet"] = "Cube Chain Mainnet";
    ChainName["CubeChainTestnet"] = "Cube Chain Testnet";
    ChainName["Teslafunds"] = "Teslafunds";
    ChainName["BONNetwork"] = "BON Network";
    ChainName["EurusTestnet"] = "Eurus Testnet";
    ChainName["EtherGem"] = "EtherGem";
    ChainName["MilkomedaC1Mainnet"] = "Milkomeda C1 Mainnet";
    ChainName["CloudWalkTestnet"] = "CloudWalk Testnet";
    ChainName["CloudWalkMainnet"] = "CloudWalk Mainnet";
    ChainName["420coin"] = "420coin";
    ChainName["EdgewareMainnet"] = "Edgeware Mainnet";
    ChainName["BeresheetTestnet"] = "Beresheet Testnet";
    ChainName["TaycanTestnet"] = "Taycan Testnet";
    ChainName["RangersProtocolMainnet"] = "Rangers Protocol Mainnet";
    ChainName["EcoballMainnet"] = "Ecoball Mainnet";
    ChainName["EcoballTestnetEspuma"] = "Ecoball Testnet Espuma";
    ChainName["FindoraMainnet"] = "Findora Mainnet";
    ChainName["FindoraTestnet"] = "Findora Testnet";
    ChainName["EvanescoMainnet"] = "Evanesco Mainnet";
    ChainName["KavaEVMTestnet"] = "Kava EVM Testnet";
    ChainName["KavaEVM"] = "Kava EVM";
    ChainName["VChainMainnet"] = "VChain Mainnet";
    ChainName["KorthoMainnet"] = "Kortho Mainnet";
    ChainName["TechPayMainnet"] = "TechPay Mainnet";
    ChainName["EZChainC-ChainMainnet"] = "EZChain C-Chain Mainnet";
    ChainName["EZChainC-ChainTestnet"] = "EZChain C-Chain Testnet";
    ChainName["CENNZnetRata"] = "CENNZnet Rata";
    ChainName["CENNZnetNikau"] = "CENNZnet Nikau";
    ChainName["ZCoreTestnet"] = "ZCore Testnet";
    ChainName["Web3QTestnet"] = "Web3Q Testnet";
    ChainName["Web3QGalileo"] = "Web3Q Galileo";
    ChainName["ParibuNetMainnet"] = "Paribu Net Mainnet";
    ChainName["ParibuNetTestnet"] = "Paribu Net Testnet";
    ChainName["BittexMainnet"] = "Bittex Mainnet";
    ChainName["Crossbell"] = "Crossbell";
    ChainName["DYNOMainnet"] = "DYNO Mainnet";
    ChainName["DYNOTestnet"] = "DYNO Testnet";
    ChainName["YuanChainMainnet"] = "YuanChain Mainnet";
    ChainName["FantomTestnet"] = "Fantom Testnet";
    ChainName["AIOZNetworkTestnet"] = "AIOZ Network Testnet";
    ChainName["PHINetwork"] = "PHI Network";
    ChainName["IoTeXNetworkMainnet"] = "IoTeX Network Mainnet";
    ChainName["IoTeXNetworkTestnet"] = "IoTeX Network Testnet";
    ChainName["VenidiumTestnet"] = "Venidium Testnet";
    ChainName["VenidiumMainnet"] = "Venidium Mainnet";
    ChainName["TLChainNetworkMainnet"] = "TLChain Network Mainnet";
    ChainName["EraSwapMainnet"] = "EraSwap Mainnet";
    ChainName["UzmiNetworkMainnet"] = "Uzmi Network Mainnet";
    ChainName["NahmiiMainnet"] = "Nahmii Mainnet";
    ChainName["NahmiiTestnet"] = "Nahmii Testnet";
    ChainName["SyscoinTanenbaumTestnet"] = "Syscoin Tanenbaum Testnet";
    ChainName["DigestSwarmChain"] = "Digest Swarm Chain";
    ChainName["OntologyTestnet"] = "Ontology Testnet";
    ChainName["WegochainRubidiumMainnet"] = "Wegochain Rubidium Mainnet";
    ChainName["PixieChainMainnet"] = "Pixie Chain Mainnet";
    ChainName["Ellatheheart"] = "Ella the heart";
    ChainName["ShyftMainnet"] = "Shyft Mainnet";
    ChainName["RiseoftheWarbotsTestnet"] = "Rise of the Warbots Testnet";
    ChainName["HazlorTestnet"] = "Hazlor Testnet";
    ChainName["Teleport"] = "Teleport";
    ChainName["TeleportTestnet"] = "Teleport Testnet";
    ChainName["MDGLTestnet"] = "MDGL Testnet";
    ChainName["GeneChainAdenineTestnet"] = "GeneChain Adenine Testnet";
    ChainName["KlaytnMainnetCypress"] = "Klaytn Mainnet Cypress";
    ChainName["KorthoTest"] = "KorthoTest";
    ChainName["TOOLGlobalMainnet"] = "TOOL Global Mainnet";
    ChainName["TOOLGlobalTestnet"] = "TOOL Global Testnet";
    ChainName["AmbrosChainTestnet"] = "Ambros Chain Testnet";
    ChainName["MammothMainnet"] = "Mammoth Mainnet";
    ChainName["Bloxberg"] = "bloxberg";
    ChainName["EvmosTestnet"] = "Evmos Testnet";
    ChainName["Evmos"] = "Evmos";
    ChainName["GenesisCoin"] = "Genesis Coin";
    ChainName["RangersProtocolTestnetRobin"] = "Rangers Protocol Testnet Robin";
    ChainName["MyOwnTestnet"] = "myOwn Testnet";
    ChainName["SmartBitcoinCash"] = "Smart Bitcoin Cash";
    ChainName["SmartBitcoinCashTestnet"] = "Smart Bitcoin Cash Testnet";
    ChainName["BlockchainGenesisMainnet"] = "Blockchain Genesis Mainnet";
    ChainName["CryptoCoinPay"] = "CryptoCoinPay";
    ChainName["QuadransBlockchain"] = "Quadrans Blockchain";
    ChainName["QuadransBlockchainTestnet"] = "Quadrans Blockchain Testnet";
    ChainName["WAGMI"] = "WAGMI";
    ChainName["ShyftTestnet"] = "Shyft Testnet";
    ChainName["SingularityZEROTestnet"] = "Singularity ZERO Testnet";
    ChainName["SingularityZEROMainnet"] = "Singularity ZERO Mainnet";
    ChainName["PhoenixMainnet"] = "Phoenix Mainnet";
    ChainName["MetaDotMainnet"] = "MetaDot Mainnet";
    ChainName["MetaDotTestnet"] = "MetaDot Testnet";
    ChainName["IVARChainTestnet"] = "IVAR Chain Testnet";
    ChainName["BTCIXNetwork"] = "BTCIX Network";
    ChainName["CENNZnetAzalea"] = "CENNZnet Azalea";
    ChainName["OmChainMainnet"] = "omChain Mainnet";
    ChainName["Taycan"] = "Taycan";
    ChainName["Webchain"] = "Webchain";
    ChainName["MintMe.comCoin"] = "MintMe.com Coin";
    ChainName["OasisChainMainnet"] = "OasisChain Mainnet";
    ChainName["Piecetestnet"] = "Piece testnet";
    ChainName["EthersocialNetwork"] = "Ethersocial Network";
    ChainName["GoChainTestnet"] = "GoChain Testnet";
    ChainName["BitgertMainnet"] = "Bitgert Mainnet";
    ChainName["FusionMainnet"] = "Fusion Mainnet";
    ChainName["EnergiMainnet"] = "Energi Mainnet";
    ChainName["Pegglecoin"] = "pegglecoin";
    ChainName["ArbitrumOne"] = "Arbitrum One";
    ChainName["CeloMainnet"] = "Celo Mainnet";
    ChainName["EmeraldParatimeTestnet"] = "Emerald Paratime Testnet";
    ChainName["EmeraldParatimeMainnet"] = "Emerald Paratime Mainnet";
    ChainName["Athereum"] = "Athereum";
    ChainName["AvalancheFujiTestnet"] = "Avalanche Fuji Testnet";
    ChainName["AvalancheC-Chain"] = "Avalanche C-Chain";
    ChainName["CeloAlfajoresTestnet"] = "Celo Alfajores Testnet";
    ChainName["AutobahnNetwork"] = "Autobahn Network";
    ChainName["REINetwork"] = "REI Network";
    ChainName["EnergiTestnet"] = "Energi Testnet";
    ChainName["DFKChain"] = "DFK Chain";
    ChainName["REIChainMainnet"] = "REI Chain Mainnet";
    ChainName["REIChainTestnet"] = "REI Chain Testnet";
    ChainName["ThinkiumTestnetChain0"] = "Thinkium Testnet Chain 0";
    ChainName["ThinkiumTestnetChain1"] = "Thinkium Testnet Chain 1";
    ChainName["ThinkiumTestnetChain2"] = "Thinkium Testnet Chain 2";
    ChainName["ThinkiumTestnetChain103"] = "Thinkium Testnet Chain 103";
    ChainName["CeloBaklavaTestnet"] = "Celo Baklava Testnet";
    ChainName["MultiVACMainnet"] = "MultiVAC Mainnet";
    ChainName["ECreditsMainnet"] = "eCredits Mainnet";
    ChainName["ECreditsTestnet"] = "eCredits Testnet";
    ChainName["Condrieu"] = "Condrieu";
    ChainName["ThinkiumMainnetChain0"] = "Thinkium Mainnet Chain 0";
    ChainName["ThinkiumMainnetChain1"] = "Thinkium Mainnet Chain 1";
    ChainName["ThinkiumMainnetChain2"] = "Thinkium Mainnet Chain 2";
    ChainName["ThinkiumMainnetChain103"] = "Thinkium Mainnet Chain 103";
    ChainName["PolyjuiceTestnet"] = "Polyjuice Testnet";
    ChainName["GodwokenTestnet(V1.1)"] = "Godwoken Testnet (V1.1)";
    ChainName["GodwokenMainnet"] = "Godwoken Mainnet";
    ChainName["EnergyWebVoltaTestnet"] = "Energy Web Volta Testnet";
    ChainName["MixinVirtualMachine"] = "Mixin Virtual Machine";
    ChainName["Firenzetestnetwork"] = "Firenze test network";
    ChainName["Mumbai"] = "Mumbai";
    ChainName["IVARChainMainnet"] = "IVAR Chain Mainnet";
    ChainName["UBSmartChain(testnet)"] = "UB Smart Chain(testnet)";
    ChainName["UBSmartChain"] = "UB Smart Chain";
    ChainName["QuarkChainMainnetRoot"] = "QuarkChain Mainnet Root";
    ChainName["QuarkChainMainnetShard0"] = "QuarkChain Mainnet Shard 0";
    ChainName["QuarkChainMainnetShard1"] = "QuarkChain Mainnet Shard 1";
    ChainName["QuarkChainMainnetShard2"] = "QuarkChain Mainnet Shard 2";
    ChainName["QuarkChainMainnetShard3"] = "QuarkChain Mainnet Shard 3";
    ChainName["QuarkChainMainnetShard4"] = "QuarkChain Mainnet Shard 4";
    ChainName["QuarkChainMainnetShard5"] = "QuarkChain Mainnet Shard 5";
    ChainName["QuarkChainMainnetShard6"] = "QuarkChain Mainnet Shard 6";
    ChainName["QuarkChainMainnetShard7"] = "QuarkChain Mainnet Shard 7";
    ChainName["Crystaleum"] = "Crystaleum";
    ChainName["BROChainMainnet"] = "BROChain Mainnet";
    ChainName["QuarkChainDevnetRoot"] = "QuarkChain Devnet Root";
    ChainName["QuarkChainDevnetShard0"] = "QuarkChain Devnet Shard 0";
    ChainName["QuarkChainDevnetShard1"] = "QuarkChain Devnet Shard 1";
    ChainName["QuarkChainDevnetShard2"] = "QuarkChain Devnet Shard 2";
    ChainName["QuarkChainDevnetShard3"] = "QuarkChain Devnet Shard 3";
    ChainName["QuarkChainDevnetShard4"] = "QuarkChain Devnet Shard 4";
    ChainName["QuarkChainDevnetShard5"] = "QuarkChain Devnet Shard 5";
    ChainName["QuarkChainDevnetShard6"] = "QuarkChain Devnet Shard 6";
    ChainName["QuarkChainDevnetShard7"] = "QuarkChain Devnet Shard 7";
    ChainName["MilkomedaC1Testnet"] = "Milkomeda C1 Testnet";
    ChainName["Akroma"] = "Akroma";
    ChainName["AlayaMainnet"] = "Alaya Mainnet";
    ChainName["AlayaDevTestnet"] = "Alaya Dev Testnet";
    ChainName["PlatONMainnet"] = "PlatON Mainnet";
    ChainName["HaymoTestnet"] = "Haymo Testnet";
    ChainName["ARTISsigma1"] = "ARTIS sigma1";
    ChainName["ARTISTestnettau1"] = "ARTIS Testnet tau1";
    ChainName["SocialSmartChainMainnet"] = "Social Smart Chain Mainnet";
    ChainName["PolisTestnet"] = "Polis Testnet";
    ChainName["PolisMainnet"] = "Polis Mainnet";
    ChainName["ArbitrumRinkeby"] = "Arbitrum Rinkeby";
    ChainName["DexalotTestnet"] = "Dexalot Testnet";
    ChainName["WeelinkTestnet"] = "Weelink Testnet";
    ChainName["OpenChainMainnet"] = "OpenChain Mainnet";
    ChainName["CMP-Testnet"] = "CMP-Testnet";
    ChainName["Vision-VpioneerTestChain"] = "Vision - Vpioneer Test Chain";
    ChainName["Vision-Mainnet"] = "Vision - Mainnet";
    ChainName["EluvioContentFabric"] = "Eluvio Content Fabric";
    ChainName["EthoProtocol"] = "Etho Protocol";
    ChainName["Xerom"] = "Xerom";
    ChainName["Kintsugi"] = "Kintsugi";
    ChainName["Kiln"] = "Kiln";
    ChainName["PlatONDevTestnet"] = "PlatON Dev Testnet";
    ChainName["PlatONDevTestnet2"] = "PlatON Dev Testnet2";
    ChainName["Musicoin"] = "Musicoin";
    ChainName["Sepolia"] = "Sepolia";
    ChainName["PepChainChurchill"] = "PepChain Churchill";
    ChainName["IOLite"] = "IOLite";
    ChainName["SmartMeshMainnet"] = "SmartMesh Mainnet";
    ChainName["Quarkblockchain"] = "quarkblockchain";
    ChainName["AuxiliumNetworkMainnet"] = "Auxilium Network Mainnet";
    ChainName["JoysDigitalMainnet"] = "Joys Digital Mainnet";
    ChainName["Aquachain"] = "Aquachain";
    ChainName["JoysDigitalTestNet"] = "Joys Digital TestNet";
    ChainName["GatherMainnetNetwork"] = "Gather Mainnet Network";
    ChainName["NeonEVMDevNet"] = "Neon EVM DevNet";
    ChainName["NeonEVMMainNet"] = "Neon EVM MainNet";
    ChainName["NeonEVMTestNet"] = "Neon EVM TestNet";
    ChainName["OneLedgerMainnet"] = "OneLedger Mainnet";
    ChainName["GatherTestnetNetwork"] = "Gather Testnet Network";
    ChainName["GatherDevnetNetwork"] = "Gather Devnet Network";
    ChainName["IPOSNetwork"] = "IPOS Network";
    ChainName["AuroraMainnet"] = "Aurora Mainnet";
    ChainName["AuroraTestnet"] = "Aurora Testnet";
    ChainName["AuroraBetanet"] = "Aurora Betanet";
    ChainName["HarmonyMainnetShard0"] = "Harmony Mainnet Shard 0";
    ChainName["HarmonyMainnetShard1"] = "Harmony Mainnet Shard 1";
    ChainName["HarmonyMainnetShard2"] = "Harmony Mainnet Shard 2";
    ChainName["HarmonyMainnetShard3"] = "Harmony Mainnet Shard 3";
    ChainName["HarmonyTestnetShard0"] = "Harmony Testnet Shard 0";
    ChainName["HarmonyTestnetShard1"] = "Harmony Testnet Shard 1";
    ChainName["HarmonyTestnetShard2"] = "Harmony Testnet Shard 2";
    ChainName["HarmonyTestnetShard3"] = "Harmony Testnet Shard 3";
    ChainName["DataHopper"] = "DataHopper";
    ChainName["Pirl"] = "Pirl";
    ChainName["OneLedgerTestnetFrankenstein"] = "OneLedger Testnet Frankenstein";
    ChainName["PalmTestnet"] = "Palm Testnet";
    ChainName["Palm"] = "Palm";
    ChainName["NtityMainnet"] = "Ntity Mainnet";
    ChainName["HaradevTestnet"] = "Haradev Testnet";
    ChainName["MolereumNetwork"] = "Molereum Network";
    ChainName["GodwokenTestnet(V1)"] = "Godwoken Testnet (V1)";
})(ChainName = exports.ChainName || (exports.ChainName = {}));
var ChainId;
(function (ChainId) {
    ChainId[ChainId["EthereumMainnet"] = 1] = "EthereumMainnet";
    ChainId[ChainId["ExpanseNetwork"] = 2] = "ExpanseNetwork";
    ChainId[ChainId["Ropsten"] = 3] = "Ropsten";
    ChainId[ChainId["Rinkeby"] = 4] = "Rinkeby";
    ChainId[ChainId["Goerli"] = 5] = "Goerli";
    ChainId[ChainId["EthereumClassicTestnetKotti"] = 6] = "EthereumClassicTestnetKotti";
    ChainId[ChainId["ThaiChain"] = 7] = "ThaiChain";
    ChainId[ChainId["Ubiq"] = 8] = "Ubiq";
    ChainId[ChainId["UbiqNetworkTestnet"] = 9] = "UbiqNetworkTestnet";
    ChainId[ChainId["Optimism"] = 10] = "Optimism";
    ChainId[ChainId["MetadiumMainnet"] = 11] = "MetadiumMainnet";
    ChainId[ChainId["MetadiumTestnet"] = 12] = "MetadiumTestnet";
    ChainId[ChainId["DiodeTestnetStaging"] = 13] = "DiodeTestnetStaging";
    ChainId[ChainId["FlareMainnet"] = 14] = "FlareMainnet";
    ChainId[ChainId["DiodePrenet"] = 15] = "DiodePrenet";
    ChainId[ChainId["FlareTestnetCoston"] = 16] = "FlareTestnetCoston";
    ChainId[ChainId["ThaiChain2.0ThaiFi"] = 17] = "ThaiChain2.0ThaiFi";
    ChainId[ChainId["ThunderCoreTestnet"] = 18] = "ThunderCoreTestnet";
    ChainId[ChainId["SongbirdCanary-Network"] = 19] = "SongbirdCanary-Network";
    ChainId[ChainId["ElastosSmartChain"] = 20] = "ElastosSmartChain";
    ChainId[ChainId["ELA-ETH-SidechainTestnet"] = 21] = "ELA-ETH-SidechainTestnet";
    ChainId[ChainId["ELA-DID-SidechainMainnet"] = 22] = "ELA-DID-SidechainMainnet";
    ChainId[ChainId["ELA-DID-SidechainTestnet"] = 23] = "ELA-DID-SidechainTestnet";
    ChainId[ChainId["DithereumMainnet"] = 24] = "DithereumMainnet";
    ChainId[ChainId["CronosMainnetBeta"] = 25] = "CronosMainnetBeta";
    ChainId[ChainId["GenesisL1testnet"] = 26] = "GenesisL1testnet";
    ChainId[ChainId["ShibaChain"] = 27] = "ShibaChain";
    ChainId[ChainId["BobaNetworkRinkebyTestnet"] = 28] = "BobaNetworkRinkebyTestnet";
    ChainId[ChainId["GenesisL1"] = 29] = "GenesisL1";
    ChainId[ChainId["RSKMainnet"] = 30] = "RSKMainnet";
    ChainId[ChainId["RSKTestnet"] = 31] = "RSKTestnet";
    ChainId[ChainId["GoodDataTestnet"] = 32] = "GoodDataTestnet";
    ChainId[ChainId["GoodDataMainnet"] = 33] = "GoodDataMainnet";
    ChainId[ChainId["DithereumTestnet"] = 34] = "DithereumTestnet";
    ChainId[ChainId["TBWGChain"] = 35] = "TBWGChain";
    ChainId[ChainId["Valorbit"] = 38] = "Valorbit";
    ChainId[ChainId["TelosEVMMainnet"] = 40] = "TelosEVMMainnet";
    ChainId[ChainId["TelosEVMTestnet"] = 41] = "TelosEVMTestnet";
    ChainId[ChainId["Kovan"] = 42] = "Kovan";
    ChainId[ChainId["DarwiniaPangolinTestnet"] = 43] = "DarwiniaPangolinTestnet";
    ChainId[ChainId["DarwiniaCrabNetwork"] = 44] = "DarwiniaCrabNetwork";
    ChainId[ChainId["DarwiniaPangoroTestnet"] = 45] = "DarwiniaPangoroTestnet";
    ChainId[ChainId["XinFinNetworkMainnet"] = 50] = "XinFinNetworkMainnet";
    ChainId[ChainId["XinFinApothemTestnet"] = 51] = "XinFinApothemTestnet";
    ChainId[ChainId["CoinExSmartChainMainnet"] = 52] = "CoinExSmartChainMainnet";
    ChainId[ChainId["CoinExSmartChainTestnet"] = 53] = "CoinExSmartChainTestnet";
    ChainId[ChainId["OpenpieceMainnet"] = 54] = "OpenpieceMainnet";
    ChainId[ChainId["ZyxMainnet"] = 55] = "ZyxMainnet";
    ChainId[ChainId["BinanceSmartChainMainnet"] = 56] = "BinanceSmartChainMainnet";
    ChainId[ChainId["SyscoinMainnet"] = 57] = "SyscoinMainnet";
    ChainId[ChainId["OntologyMainnet"] = 58] = "OntologyMainnet";
    ChainId[ChainId["EOSMainnet"] = 59] = "EOSMainnet";
    ChainId[ChainId["GoChain"] = 60] = "GoChain";
    ChainId[ChainId["EthereumClassicMainnet"] = 61] = "EthereumClassicMainnet";
    ChainId[ChainId["EthereumClassicTestnetMorden"] = 62] = "EthereumClassicTestnetMorden";
    ChainId[ChainId["EthereumClassicTestnetMordor"] = 63] = "EthereumClassicTestnetMordor";
    ChainId[ChainId["Ellaism"] = 64] = "Ellaism";
    ChainId[ChainId["OKExChainTestnet"] = 65] = "OKExChainTestnet";
    ChainId[ChainId["OKXChainMainnet"] = 66] = "OKXChainMainnet";
    ChainId[ChainId["DBChainTestnet"] = 67] = "DBChainTestnet";
    ChainId[ChainId["SoterOneMainnet"] = 68] = "SoterOneMainnet";
    ChainId[ChainId["OptimismKovan"] = 69] = "OptimismKovan";
    ChainId[ChainId["HooSmartChain"] = 70] = "HooSmartChain";
    ChainId[ChainId["ConfluxeSpace(Testnet)"] = 71] = "ConfluxeSpace(Testnet)";
    ChainId[ChainId["IDChainMainnet"] = 74] = "IDChainMainnet";
    ChainId[ChainId["Mix"] = 76] = "Mix";
    ChainId[ChainId["POANetworkSokol"] = 77] = "POANetworkSokol";
    ChainId[ChainId["PrimusChainmainnet"] = 78] = "PrimusChainmainnet";
    ChainId[ChainId["ZenithMainnet"] = 79] = "ZenithMainnet";
    ChainId[ChainId["GeneChain"] = 80] = "GeneChain";
    ChainId[ChainId["ZenithTestnet(Vilnius)"] = 81] = "ZenithTestnet(Vilnius)";
    ChainId[ChainId["MeterMainnet"] = 82] = "MeterMainnet";
    ChainId[ChainId["MeterTestnet"] = 83] = "MeterTestnet";
    ChainId[ChainId["GateChainTestnet"] = 85] = "GateChainTestnet";
    ChainId[ChainId["GateChainMainnet"] = 86] = "GateChainMainnet";
    ChainId[ChainId["NovaNetwork"] = 87] = "NovaNetwork";
    ChainId[ChainId["TomoChain"] = 88] = "TomoChain";
    ChainId[ChainId["TomoChainTestnet"] = 89] = "TomoChainTestnet";
    ChainId[ChainId["GarizonStage0"] = 90] = "GarizonStage0";
    ChainId[ChainId["GarizonStage1"] = 91] = "GarizonStage1";
    ChainId[ChainId["GarizonStage2"] = 92] = "GarizonStage2";
    ChainId[ChainId["GarizonStage3"] = 93] = "GarizonStage3";
    ChainId[ChainId["CryptoKylinTestnet"] = 95] = "CryptoKylinTestnet";
    ChainId[ChainId["NEXTSmartChain"] = 96] = "NEXTSmartChain";
    ChainId[ChainId["BinanceSmartChainTestnet"] = 97] = "BinanceSmartChainTestnet";
    ChainId[ChainId["POANetworkCore"] = 99] = "POANetworkCore";
    ChainId[ChainId["Gnosis"] = 100] = "Gnosis";
    ChainId[ChainId["EtherInc"] = 101] = "EtherInc";
    ChainId[ChainId["Web3GamesTestnet"] = 102] = "Web3GamesTestnet";
    ChainId[ChainId["Web3GamesDevnet"] = 105] = "Web3GamesDevnet";
    ChainId[ChainId["VelasEVMMainnet"] = 106] = "VelasEVMMainnet";
    ChainId[ChainId["NebulaTestnet"] = 107] = "NebulaTestnet";
    ChainId[ChainId["ThunderCoreMainnet"] = 108] = "ThunderCoreMainnet";
    ChainId[ChainId["ProtonTestnet"] = 110] = "ProtonTestnet";
    ChainId[ChainId["EtherLiteChain"] = 111] = "EtherLiteChain";
    ChainId[ChainId["FuseMainnet"] = 122] = "FuseMainnet";
    ChainId[ChainId["FuseSparknet"] = 123] = "FuseSparknet";
    ChainId[ChainId["DecentralizedWebMainnet"] = 124] = "DecentralizedWebMainnet";
    ChainId[ChainId["OYchainTestnet"] = 125] = "OYchainTestnet";
    ChainId[ChainId["OYchainMainnet"] = 126] = "OYchainMainnet";
    ChainId[ChainId["Factory127Mainnet"] = 127] = "Factory127Mainnet";
    ChainId[ChainId["HuobiECOChainMainnet"] = 128] = "HuobiECOChainMainnet";
    ChainId[ChainId["PolygonMainnet"] = 137] = "PolygonMainnet";
    ChainId[ChainId["OpenpieceTestnet"] = 141] = "OpenpieceTestnet";
    ChainId[ChainId["DAXCHAIN"] = 142] = "DAXCHAIN";
    ChainId[ChainId["LightstreamsTestnet"] = 162] = "LightstreamsTestnet";
    ChainId[ChainId["LightstreamsMainnet"] = 163] = "LightstreamsMainnet";
    ChainId[ChainId["AIOZNetwork"] = 168] = "AIOZNetwork";
    ChainId[ChainId["HOOSmartChainTestnet"] = 170] = "HOOSmartChainTestnet";
    ChainId[ChainId["Latam-BlockchainResilTestnet"] = 172] = "Latam-BlockchainResilTestnet";
    ChainId[ChainId["AMEChainMainnet"] = 180] = "AMEChainMainnet";
    ChainId[ChainId["SeeleMainnet"] = 186] = "SeeleMainnet";
    ChainId[ChainId["BMCMainnet"] = 188] = "BMCMainnet";
    ChainId[ChainId["BMCTestnet"] = 189] = "BMCTestnet";
    ChainId[ChainId["CryptoEmergency"] = 193] = "CryptoEmergency";
    ChainId[ChainId["BitTorrentChainMainnet"] = 199] = "BitTorrentChainMainnet";
    ChainId[ChainId["ArbitrumonxDai"] = 200] = "ArbitrumonxDai";
    ChainId[ChainId["FreightTrustNetwork"] = 211] = "FreightTrustNetwork";
    ChainId[ChainId["SoterOneMainnetold"] = 218] = "SoterOneMainnetold";
    ChainId[ChainId["Permission"] = 222] = "Permission";
    ChainId[ChainId["LACHAINMainnet"] = 225] = "LACHAINMainnet";
    ChainId[ChainId["LACHAINTestnet"] = 226] = "LACHAINTestnet";
    ChainId[ChainId["AitdTestnet"] = 239] = "AitdTestnet";
    ChainId[ChainId["EnergyWebChain"] = 246] = "EnergyWebChain";
    ChainId[ChainId["FantomOpera"] = 250] = "FantomOpera";
    ChainId[ChainId["HuobiECOChainTestnet"] = 256] = "HuobiECOChainTestnet";
    ChainId[ChainId["Setheum"] = 258] = "Setheum";
    ChainId[ChainId["SURBlockchainNetwork"] = 262] = "SURBlockchainNetwork";
    ChainId[ChainId["HighPerformanceBlockchain"] = 269] = "HighPerformanceBlockchain";
    ChainId[ChainId["BobaNetwork"] = 288] = "BobaNetwork";
    ChainId[ChainId["OptimismonGnosisChain"] = 300] = "OptimismonGnosisChain";
    ChainId[ChainId["KCCMainnet"] = 321] = "KCCMainnet";
    ChainId[ChainId["KCCTestnet"] = 322] = "KCCTestnet";
    ChainId[ChainId["Web3QMainnet"] = 333] = "Web3QMainnet";
    ChainId[ChainId["DFKChainTest"] = 335] = "DFKChainTest";
    ChainId[ChainId["Shiden"] = 336] = "Shiden";
    ChainId[ChainId["CronosTestnet"] = 338] = "CronosTestnet";
    ChainId[ChainId["ThetaMainnet"] = 361] = "ThetaMainnet";
    ChainId[ChainId["ThetaSapphireTestnet"] = 363] = "ThetaSapphireTestnet";
    ChainId[ChainId["ThetaAmberTestnet"] = 364] = "ThetaAmberTestnet";
    ChainId[ChainId["ThetaTestnet"] = 365] = "ThetaTestnet";
    ChainId[ChainId["PulseChainMainnet"] = 369] = "PulseChainMainnet";
    ChainId[ChainId["Lisinski"] = 385] = "Lisinski";
    ChainId[ChainId["SXNetworkMainnet"] = 416] = "SXNetworkMainnet";
    ChainId[ChainId["OptimisticEthereumTestnetGoerli"] = 420] = "OptimisticEthereumTestnetGoerli";
    ChainId[ChainId["Rupaya"] = 499] = "Rupaya";
    ChainId[ChainId["Double-AChainMainnet"] = 512] = "Double-AChainMainnet";
    ChainId[ChainId["Double-AChainTestnet"] = 513] = "Double-AChainTestnet";
    ChainId[ChainId["XTSmartChainMainnet"] = 520] = "XTSmartChainMainnet";
    ChainId[ChainId["Candle"] = 534] = "Candle";
    ChainId[ChainId["Vela1ChainMainnet"] = 555] = "Vela1ChainMainnet";
    ChainId[ChainId["TaoNetwork"] = 558] = "TaoNetwork";
    ChainId[ChainId["MetisStardustTestnet"] = 588] = "MetisStardustTestnet";
    ChainId[ChainId["Astar"] = 592] = "Astar";
    ChainId[ChainId["AcalaMandalaTestnet"] = 595] = "AcalaMandalaTestnet";
    ChainId[ChainId["KaruraNetworkTestnet"] = 596] = "KaruraNetworkTestnet";
    ChainId[ChainId["AcalaNetworkTestnet"] = 597] = "AcalaNetworkTestnet";
    ChainId[ChainId["Meshnyantestnet"] = 600] = "Meshnyantestnet";
    ChainId[ChainId["SXNetworkTestnet"] = 647] = "SXNetworkTestnet";
    ChainId[ChainId["PixieChainTestnet"] = 666] = "PixieChainTestnet";
    ChainId[ChainId["KaruraNetwork"] = 686] = "KaruraNetwork";
    ChainId[ChainId["StarSocialTestnet"] = 700] = "StarSocialTestnet";
    ChainId[ChainId["BlockChainStationMainnet"] = 707] = "BlockChainStationMainnet";
    ChainId[ChainId["BlockChainStationTestnet"] = 708] = "BlockChainStationTestnet";
    ChainId[ChainId["Factory127Testnet"] = 721] = "Factory127Testnet";
    ChainId[ChainId["OpenChainTestnet"] = 776] = "OpenChainTestnet";
    ChainId[ChainId["CheapETH"] = 777] = "CheapETH";
    ChainId[ChainId["AcalaNetwork"] = 787] = "AcalaNetwork";
    ChainId[ChainId["AerochainTestnet"] = 788] = "AerochainTestnet";
    ChainId[ChainId["Haic"] = 803] = "Haic";
    ChainId[ChainId["CallistoMainnet"] = 820] = "CallistoMainnet";
    ChainId[ChainId["CallistoTestnet"] = 821] = "CallistoTestnet";
    ChainId[ChainId["AmbrosChainMainnet"] = 880] = "AmbrosChainMainnet";
    ChainId[ChainId["Wanchain"] = 888] = "Wanchain";
    ChainId[ChainId["GarizonTestnetStage0"] = 900] = "GarizonTestnetStage0";
    ChainId[ChainId["GarizonTestnetStage1"] = 901] = "GarizonTestnetStage1";
    ChainId[ChainId["GarizonTestnetStage2"] = 902] = "GarizonTestnetStage2";
    ChainId[ChainId["GarizonTestnetStage3"] = 903] = "GarizonTestnetStage3";
    ChainId[ChainId["PulseChainTestnet"] = 940] = "PulseChainTestnet";
    ChainId[ChainId["PulseChainTestnetv2b"] = 941] = "PulseChainTestnetv2b";
    ChainId[ChainId["PulseChainTestnetv3"] = 942] = "PulseChainTestnetv3";
    ChainId[ChainId["NepalBlockchainNetwork"] = 977] = "NepalBlockchainNetwork";
    ChainId[ChainId["TOPMainnetEVM"] = 980] = "TOPMainnetEVM";
    ChainId[ChainId["TOPMainnet"] = 989] = "TOPMainnet";
    ChainId[ChainId["LuckyNetwork"] = 998] = "LuckyNetwork";
    ChainId[ChainId["WanchainTestnet"] = 999] = "WanchainTestnet";
    ChainId[ChainId["KlaytnTestnetBaobab"] = 1001] = "KlaytnTestnetBaobab";
    ChainId[ChainId["NewtonTestnet"] = 1007] = "NewtonTestnet";
    ChainId[ChainId["EurusMainnet"] = 1008] = "EurusMainnet";
    ChainId[ChainId["EvriceNetwork"] = 1010] = "EvriceNetwork";
    ChainId[ChainId["Newton"] = 1012] = "Newton";
    ChainId[ChainId["Sakura"] = 1022] = "Sakura";
    ChainId[ChainId["CloverTestnet"] = 1023] = "CloverTestnet";
    ChainId[ChainId["CLVParachain"] = 1024] = "CLVParachain";
    ChainId[ChainId["BitTorrentChainTestnet"] = 1028] = "BitTorrentChainTestnet";
    ChainId[ChainId["ConfluxeSpace"] = 1030] = "ConfluxeSpace";
    ChainId[ChainId["MetisAndromedaMainnet"] = 1088] = "MetisAndromedaMainnet";
    ChainId[ChainId["MathChain"] = 1139] = "MathChain";
    ChainId[ChainId["MathChainTestnet"] = 1140] = "MathChainTestnet";
    ChainId[ChainId["IoraChain"] = 1197] = "IoraChain";
    ChainId[ChainId["EvanescoTestnet"] = 1201] = "EvanescoTestnet";
    ChainId[ChainId["WorldTradeTechnicalChainMainnet"] = 1202] = "WorldTradeTechnicalChainMainnet";
    ChainId[ChainId["PopcateumMainnet"] = 1213] = "PopcateumMainnet";
    ChainId[ChainId["EnterChainMainnet"] = 1214] = "EnterChainMainnet";
    ChainId[ChainId["HALOMainnet"] = 1280] = "HALOMainnet";
    ChainId[ChainId["Moonbeam"] = 1284] = "Moonbeam";
    ChainId[ChainId["Moonriver"] = 1285] = "Moonriver";
    ChainId[ChainId["Moonrockold"] = 1286] = "Moonrockold";
    ChainId[ChainId["MoonbaseAlpha"] = 1287] = "MoonbaseAlpha";
    ChainId[ChainId["Moonrock"] = 1288] = "Moonrock";
    ChainId[ChainId["CENNZnetold"] = 1337] = "CENNZnetold";
    ChainId[ChainId["SherpaxMainnet"] = 1506] = "SherpaxMainnet";
    ChainId[ChainId["SherpaxTestnet"] = 1507] = "SherpaxTestnet";
    ChainId[ChainId["CatecoinChainMainnet"] = 1618] = "CatecoinChainMainnet";
    ChainId[ChainId["Atheios"] = 1620] = "Atheios";
    ChainId[ChainId["Btachain"] = 1657] = "Btachain";
    ChainId[ChainId["LUDANMainnet"] = 1688] = "LUDANMainnet";
    ChainId[ChainId["CubeChainMainnet"] = 1818] = "CubeChainMainnet";
    ChainId[ChainId["CubeChainTestnet"] = 1819] = "CubeChainTestnet";
    ChainId[ChainId["Teslafunds"] = 1856] = "Teslafunds";
    ChainId[ChainId["BONNetwork"] = 1898] = "BONNetwork";
    ChainId[ChainId["EurusTestnet"] = 1984] = "EurusTestnet";
    ChainId[ChainId["EtherGem"] = 1987] = "EtherGem";
    ChainId[ChainId["MilkomedaC1Mainnet"] = 2001] = "MilkomedaC1Mainnet";
    ChainId[ChainId["CloudWalkTestnet"] = 2008] = "CloudWalkTestnet";
    ChainId[ChainId["CloudWalkMainnet"] = 2009] = "CloudWalkMainnet";
    ChainId[ChainId["420coin"] = 2020] = "420coin";
    ChainId[ChainId["EdgewareMainnet"] = 2021] = "EdgewareMainnet";
    ChainId[ChainId["BeresheetTestnet"] = 2022] = "BeresheetTestnet";
    ChainId[ChainId["TaycanTestnet"] = 2023] = "TaycanTestnet";
    ChainId[ChainId["RangersProtocolMainnet"] = 2025] = "RangersProtocolMainnet";
    ChainId[ChainId["EcoballMainnet"] = 2100] = "EcoballMainnet";
    ChainId[ChainId["EcoballTestnetEspuma"] = 2101] = "EcoballTestnetEspuma";
    ChainId[ChainId["FindoraMainnet"] = 2152] = "FindoraMainnet";
    ChainId[ChainId["FindoraTestnet"] = 2153] = "FindoraTestnet";
    ChainId[ChainId["EvanescoMainnet"] = 2213] = "EvanescoMainnet";
    ChainId[ChainId["KavaEVMTestnet"] = 2221] = "KavaEVMTestnet";
    ChainId[ChainId["KavaEVM"] = 2222] = "KavaEVM";
    ChainId[ChainId["VChainMainnet"] = 2223] = "VChainMainnet";
    ChainId[ChainId["KorthoMainnet"] = 2559] = "KorthoMainnet";
    ChainId[ChainId["TechPayMainnet"] = 2569] = "TechPayMainnet";
    ChainId[ChainId["EZChainC-ChainMainnet"] = 2612] = "EZChainC-ChainMainnet";
    ChainId[ChainId["EZChainC-ChainTestnet"] = 2613] = "EZChainC-ChainTestnet";
    ChainId[ChainId["CENNZnetRata"] = 3000] = "CENNZnetRata";
    ChainId[ChainId["CENNZnetNikau"] = 3001] = "CENNZnetNikau";
    ChainId[ChainId["ZCoreTestnet"] = 3331] = "ZCoreTestnet";
    ChainId[ChainId["Web3QTestnet"] = 3333] = "Web3QTestnet";
    ChainId[ChainId["Web3QGalileo"] = 3334] = "Web3QGalileo";
    ChainId[ChainId["ParibuNetMainnet"] = 3400] = "ParibuNetMainnet";
    ChainId[ChainId["ParibuNetTestnet"] = 3500] = "ParibuNetTestnet";
    ChainId[ChainId["BittexMainnet"] = 3690] = "BittexMainnet";
    ChainId[ChainId["Crossbell"] = 3737] = "Crossbell";
    ChainId[ChainId["DYNOMainnet"] = 3966] = "DYNOMainnet";
    ChainId[ChainId["DYNOTestnet"] = 3967] = "DYNOTestnet";
    ChainId[ChainId["YuanChainMainnet"] = 3999] = "YuanChainMainnet";
    ChainId[ChainId["FantomTestnet"] = 4002] = "FantomTestnet";
    ChainId[ChainId["AIOZNetworkTestnet"] = 4102] = "AIOZNetworkTestnet";
    ChainId[ChainId["PHINetwork"] = 4181] = "PHINetwork";
    ChainId[ChainId["IoTeXNetworkMainnet"] = 4689] = "IoTeXNetworkMainnet";
    ChainId[ChainId["IoTeXNetworkTestnet"] = 4690] = "IoTeXNetworkTestnet";
    ChainId[ChainId["VenidiumTestnet"] = 4918] = "VenidiumTestnet";
    ChainId[ChainId["VenidiumMainnet"] = 4919] = "VenidiumMainnet";
    ChainId[ChainId["TLChainNetworkMainnet"] = 5177] = "TLChainNetworkMainnet";
    ChainId[ChainId["EraSwapMainnet"] = 5197] = "EraSwapMainnet";
    ChainId[ChainId["UzmiNetworkMainnet"] = 5315] = "UzmiNetworkMainnet";
    ChainId[ChainId["NahmiiMainnet"] = 5551] = "NahmiiMainnet";
    ChainId[ChainId["NahmiiTestnet"] = 5553] = "NahmiiTestnet";
    ChainId[ChainId["SyscoinTanenbaumTestnet"] = 5700] = "SyscoinTanenbaumTestnet";
    ChainId[ChainId["DigestSwarmChain"] = 5777] = "DigestSwarmChain";
    ChainId[ChainId["OntologyTestnet"] = 5851] = "OntologyTestnet";
    ChainId[ChainId["WegochainRubidiumMainnet"] = 5869] = "WegochainRubidiumMainnet";
    ChainId[ChainId["PixieChainMainnet"] = 6626] = "PixieChainMainnet";
    ChainId[ChainId["Ellatheheart"] = 7027] = "Ellatheheart";
    ChainId[ChainId["ShyftMainnet"] = 7341] = "ShyftMainnet";
    ChainId[ChainId["RiseoftheWarbotsTestnet"] = 7777] = "RiseoftheWarbotsTestnet";
    ChainId[ChainId["HazlorTestnet"] = 7878] = "HazlorTestnet";
    ChainId[ChainId["Teleport"] = 8000] = "Teleport";
    ChainId[ChainId["TeleportTestnet"] = 8001] = "TeleportTestnet";
    ChainId[ChainId["MDGLTestnet"] = 8029] = "MDGLTestnet";
    ChainId[ChainId["GeneChainAdenineTestnet"] = 8080] = "GeneChainAdenineTestnet";
    ChainId[ChainId["KlaytnMainnetCypress"] = 8217] = "KlaytnMainnetCypress";
    ChainId[ChainId["KorthoTest"] = 8285] = "KorthoTest";
    ChainId[ChainId["TOOLGlobalMainnet"] = 8723] = "TOOLGlobalMainnet";
    ChainId[ChainId["TOOLGlobalTestnet"] = 8724] = "TOOLGlobalTestnet";
    ChainId[ChainId["AmbrosChainTestnet"] = 8888] = "AmbrosChainTestnet";
    ChainId[ChainId["MammothMainnet"] = 8898] = "MammothMainnet";
    ChainId[ChainId["Bloxberg"] = 8995] = "Bloxberg";
    ChainId[ChainId["EvmosTestnet"] = 9000] = "EvmosTestnet";
    ChainId[ChainId["Evmos"] = 9001] = "Evmos";
    ChainId[ChainId["GenesisCoin"] = 9100] = "GenesisCoin";
    ChainId[ChainId["RangersProtocolTestnetRobin"] = 9527] = "RangersProtocolTestnetRobin";
    ChainId[ChainId["MyOwnTestnet"] = 9999] = "MyOwnTestnet";
    ChainId[ChainId["SmartBitcoinCash"] = 10000] = "SmartBitcoinCash";
    ChainId[ChainId["SmartBitcoinCashTestnet"] = 10001] = "SmartBitcoinCashTestnet";
    ChainId[ChainId["BlockchainGenesisMainnet"] = 10101] = "BlockchainGenesisMainnet";
    ChainId[ChainId["CryptoCoinPay"] = 10823] = "CryptoCoinPay";
    ChainId[ChainId["QuadransBlockchain"] = 10946] = "QuadransBlockchain";
    ChainId[ChainId["QuadransBlockchainTestnet"] = 10947] = "QuadransBlockchainTestnet";
    ChainId[ChainId["WAGMI"] = 11111] = "WAGMI";
    ChainId[ChainId["ShyftTestnet"] = 11437] = "ShyftTestnet";
    ChainId[ChainId["SingularityZEROTestnet"] = 12051] = "SingularityZEROTestnet";
    ChainId[ChainId["SingularityZEROMainnet"] = 12052] = "SingularityZEROMainnet";
    ChainId[ChainId["PhoenixMainnet"] = 13381] = "PhoenixMainnet";
    ChainId[ChainId["MetaDotMainnet"] = 16000] = "MetaDotMainnet";
    ChainId[ChainId["MetaDotTestnet"] = 16001] = "MetaDotTestnet";
    ChainId[ChainId["IVARChainTestnet"] = 16888] = "IVARChainTestnet";
    ChainId[ChainId["BTCIXNetwork"] = 19845] = "BTCIXNetwork";
    ChainId[ChainId["CENNZnetAzalea"] = 21337] = "CENNZnetAzalea";
    ChainId[ChainId["OmChainMainnet"] = 21816] = "OmChainMainnet";
    ChainId[ChainId["Taycan"] = 22023] = "Taycan";
    ChainId[ChainId["Webchain"] = 24484] = "Webchain";
    ChainId[ChainId["MintMe.comCoin"] = 24734] = "MintMe.comCoin";
    ChainId[ChainId["OasisChainMainnet"] = 26863] = "OasisChainMainnet";
    ChainId[ChainId["Piecetestnet"] = 30067] = "Piecetestnet";
    ChainId[ChainId["EthersocialNetwork"] = 31102] = "EthersocialNetwork";
    ChainId[ChainId["GoChainTestnet"] = 31337] = "GoChainTestnet";
    ChainId[ChainId["BitgertMainnet"] = 32520] = "BitgertMainnet";
    ChainId[ChainId["FusionMainnet"] = 32659] = "FusionMainnet";
    ChainId[ChainId["EnergiMainnet"] = 39797] = "EnergiMainnet";
    ChainId[ChainId["Pegglecoin"] = 42069] = "Pegglecoin";
    ChainId[ChainId["ArbitrumOne"] = 42161] = "ArbitrumOne";
    ChainId[ChainId["CeloMainnet"] = 42220] = "CeloMainnet";
    ChainId[ChainId["EmeraldParatimeTestnet"] = 42261] = "EmeraldParatimeTestnet";
    ChainId[ChainId["EmeraldParatimeMainnet"] = 42262] = "EmeraldParatimeMainnet";
    ChainId[ChainId["Athereum"] = 43110] = "Athereum";
    ChainId[ChainId["AvalancheFujiTestnet"] = 43113] = "AvalancheFujiTestnet";
    ChainId[ChainId["AvalancheC-Chain"] = 43114] = "AvalancheC-Chain";
    ChainId[ChainId["CeloAlfajoresTestnet"] = 44787] = "CeloAlfajoresTestnet";
    ChainId[ChainId["AutobahnNetwork"] = 45000] = "AutobahnNetwork";
    ChainId[ChainId["REINetwork"] = 47805] = "REINetwork";
    ChainId[ChainId["EnergiTestnet"] = 49797] = "EnergiTestnet";
    ChainId[ChainId["DFKChain"] = 53935] = "DFKChain";
    ChainId[ChainId["REIChainMainnet"] = 55555] = "REIChainMainnet";
    ChainId[ChainId["REIChainTestnet"] = 55556] = "REIChainTestnet";
    ChainId[ChainId["ThinkiumTestnetChain0"] = 60000] = "ThinkiumTestnetChain0";
    ChainId[ChainId["ThinkiumTestnetChain1"] = 60001] = "ThinkiumTestnetChain1";
    ChainId[ChainId["ThinkiumTestnetChain2"] = 60002] = "ThinkiumTestnetChain2";
    ChainId[ChainId["ThinkiumTestnetChain103"] = 60103] = "ThinkiumTestnetChain103";
    ChainId[ChainId["CeloBaklavaTestnet"] = 62320] = "CeloBaklavaTestnet";
    ChainId[ChainId["MultiVACMainnet"] = 62621] = "MultiVACMainnet";
    ChainId[ChainId["ECreditsMainnet"] = 63000] = "ECreditsMainnet";
    ChainId[ChainId["ECreditsTestnet"] = 63001] = "ECreditsTestnet";
    ChainId[ChainId["Condrieu"] = 69420] = "Condrieu";
    ChainId[ChainId["ThinkiumMainnetChain0"] = 70000] = "ThinkiumMainnetChain0";
    ChainId[ChainId["ThinkiumMainnetChain1"] = 70001] = "ThinkiumMainnetChain1";
    ChainId[ChainId["ThinkiumMainnetChain2"] = 70002] = "ThinkiumMainnetChain2";
    ChainId[ChainId["ThinkiumMainnetChain103"] = 70103] = "ThinkiumMainnetChain103";
    ChainId[ChainId["PolyjuiceTestnet"] = 71393] = "PolyjuiceTestnet";
    ChainId[ChainId["GodwokenTestnet(V1.1)"] = 71401] = "GodwokenTestnet(V1.1)";
    ChainId[ChainId["GodwokenMainnet"] = 71402] = "GodwokenMainnet";
    ChainId[ChainId["EnergyWebVoltaTestnet"] = 73799] = "EnergyWebVoltaTestnet";
    ChainId[ChainId["MixinVirtualMachine"] = 73927] = "MixinVirtualMachine";
    ChainId[ChainId["Firenzetestnetwork"] = 78110] = "Firenzetestnetwork";
    ChainId[ChainId["Mumbai"] = 80001] = "Mumbai";
    ChainId[ChainId["IVARChainMainnet"] = 88888] = "IVARChainMainnet";
    ChainId[ChainId["UBSmartChain(testnet)"] = 99998] = "UBSmartChain(testnet)";
    ChainId[ChainId["UBSmartChain"] = 99999] = "UBSmartChain";
    ChainId[ChainId["QuarkChainMainnetRoot"] = 100000] = "QuarkChainMainnetRoot";
    ChainId[ChainId["QuarkChainMainnetShard0"] = 100001] = "QuarkChainMainnetShard0";
    ChainId[ChainId["QuarkChainMainnetShard1"] = 100002] = "QuarkChainMainnetShard1";
    ChainId[ChainId["QuarkChainMainnetShard2"] = 100003] = "QuarkChainMainnetShard2";
    ChainId[ChainId["QuarkChainMainnetShard3"] = 100004] = "QuarkChainMainnetShard3";
    ChainId[ChainId["QuarkChainMainnetShard4"] = 100005] = "QuarkChainMainnetShard4";
    ChainId[ChainId["QuarkChainMainnetShard5"] = 100006] = "QuarkChainMainnetShard5";
    ChainId[ChainId["QuarkChainMainnetShard6"] = 100007] = "QuarkChainMainnetShard6";
    ChainId[ChainId["QuarkChainMainnetShard7"] = 100008] = "QuarkChainMainnetShard7";
    ChainId[ChainId["Crystaleum"] = 103090] = "Crystaleum";
    ChainId[ChainId["BROChainMainnet"] = 108801] = "BROChainMainnet";
    ChainId[ChainId["QuarkChainDevnetRoot"] = 110000] = "QuarkChainDevnetRoot";
    ChainId[ChainId["QuarkChainDevnetShard0"] = 110001] = "QuarkChainDevnetShard0";
    ChainId[ChainId["QuarkChainDevnetShard1"] = 110002] = "QuarkChainDevnetShard1";
    ChainId[ChainId["QuarkChainDevnetShard2"] = 110003] = "QuarkChainDevnetShard2";
    ChainId[ChainId["QuarkChainDevnetShard3"] = 110004] = "QuarkChainDevnetShard3";
    ChainId[ChainId["QuarkChainDevnetShard4"] = 110005] = "QuarkChainDevnetShard4";
    ChainId[ChainId["QuarkChainDevnetShard5"] = 110006] = "QuarkChainDevnetShard5";
    ChainId[ChainId["QuarkChainDevnetShard6"] = 110007] = "QuarkChainDevnetShard6";
    ChainId[ChainId["QuarkChainDevnetShard7"] = 110008] = "QuarkChainDevnetShard7";
    ChainId[ChainId["MilkomedaC1Testnet"] = 200101] = "MilkomedaC1Testnet";
    ChainId[ChainId["Akroma"] = 200625] = "Akroma";
    ChainId[ChainId["AlayaMainnet"] = 201018] = "AlayaMainnet";
    ChainId[ChainId["AlayaDevTestnet"] = 201030] = "AlayaDevTestnet";
    ChainId[ChainId["PlatONMainnet"] = 210425] = "PlatONMainnet";
    ChainId[ChainId["HaymoTestnet"] = 234666] = "HaymoTestnet";
    ChainId[ChainId["ARTISsigma1"] = 246529] = "ARTISsigma1";
    ChainId[ChainId["ARTISTestnettau1"] = 246785] = "ARTISTestnettau1";
    ChainId[ChainId["SocialSmartChainMainnet"] = 281121] = "SocialSmartChainMainnet";
    ChainId[ChainId["PolisTestnet"] = 333888] = "PolisTestnet";
    ChainId[ChainId["PolisMainnet"] = 333999] = "PolisMainnet";
    ChainId[ChainId["ArbitrumRinkeby"] = 421611] = "ArbitrumRinkeby";
    ChainId[ChainId["DexalotTestnet"] = 432201] = "DexalotTestnet";
    ChainId[ChainId["WeelinkTestnet"] = 444900] = "WeelinkTestnet";
    ChainId[ChainId["OpenChainMainnet"] = 474142] = "OpenChainMainnet";
    ChainId[ChainId["CMP-Testnet"] = 512512] = "CMP-Testnet";
    ChainId[ChainId["Vision-VpioneerTestChain"] = 666666] = "Vision-VpioneerTestChain";
    ChainId[ChainId["Vision-Mainnet"] = 888888] = "Vision-Mainnet";
    ChainId[ChainId["EluvioContentFabric"] = 955305] = "EluvioContentFabric";
    ChainId[ChainId["EthoProtocol"] = 1313114] = "EthoProtocol";
    ChainId[ChainId["Xerom"] = 1313500] = "Xerom";
    ChainId[ChainId["Kintsugi"] = 1337702] = "Kintsugi";
    ChainId[ChainId["Kiln"] = 1337802] = "Kiln";
    ChainId[ChainId["PlatONDevTestnet"] = 2203181] = "PlatONDevTestnet";
    ChainId[ChainId["PlatONDevTestnet2"] = 2206132] = "PlatONDevTestnet2";
    ChainId[ChainId["Musicoin"] = 7762959] = "Musicoin";
    ChainId[ChainId["Sepolia"] = 11155111] = "Sepolia";
    ChainId[ChainId["PepChainChurchill"] = 13371337] = "PepChainChurchill";
    ChainId[ChainId["IOLite"] = 18289463] = "IOLite";
    ChainId[ChainId["SmartMeshMainnet"] = 20180430] = "SmartMeshMainnet";
    ChainId[ChainId["Quarkblockchain"] = 20181205] = "Quarkblockchain";
    ChainId[ChainId["AuxiliumNetworkMainnet"] = 28945486] = "AuxiliumNetworkMainnet";
    ChainId[ChainId["JoysDigitalMainnet"] = 35855456] = "JoysDigitalMainnet";
    ChainId[ChainId["Aquachain"] = 61717561] = "Aquachain";
    ChainId[ChainId["JoysDigitalTestNet"] = 99415706] = "JoysDigitalTestNet";
    ChainId[ChainId["GatherMainnetNetwork"] = 192837465] = "GatherMainnetNetwork";
    ChainId[ChainId["NeonEVMDevNet"] = 245022926] = "NeonEVMDevNet";
    ChainId[ChainId["NeonEVMMainNet"] = 245022934] = "NeonEVMMainNet";
    ChainId[ChainId["NeonEVMTestNet"] = 245022940] = "NeonEVMTestNet";
    ChainId[ChainId["OneLedgerMainnet"] = 311752642] = "OneLedgerMainnet";
    ChainId[ChainId["GatherTestnetNetwork"] = 356256156] = "GatherTestnetNetwork";
    ChainId[ChainId["GatherDevnetNetwork"] = 486217935] = "GatherDevnetNetwork";
    ChainId[ChainId["IPOSNetwork"] = 1122334455] = "IPOSNetwork";
    ChainId[ChainId["AuroraMainnet"] = 1313161554] = "AuroraMainnet";
    ChainId[ChainId["AuroraTestnet"] = 1313161555] = "AuroraTestnet";
    ChainId[ChainId["AuroraBetanet"] = 1313161556] = "AuroraBetanet";
    ChainId[ChainId["HarmonyMainnetShard0"] = 1666600000] = "HarmonyMainnetShard0";
    ChainId[ChainId["HarmonyMainnetShard1"] = 1666600001] = "HarmonyMainnetShard1";
    ChainId[ChainId["HarmonyMainnetShard2"] = 1666600002] = "HarmonyMainnetShard2";
    ChainId[ChainId["HarmonyMainnetShard3"] = 1666600003] = "HarmonyMainnetShard3";
    ChainId[ChainId["HarmonyTestnetShard0"] = 1666700000] = "HarmonyTestnetShard0";
    ChainId[ChainId["HarmonyTestnetShard1"] = 1666700001] = "HarmonyTestnetShard1";
    ChainId[ChainId["HarmonyTestnetShard2"] = 1666700002] = "HarmonyTestnetShard2";
    ChainId[ChainId["HarmonyTestnetShard3"] = 1666700003] = "HarmonyTestnetShard3";
    ChainId[ChainId["DataHopper"] = 2021121117] = "DataHopper";
    ChainId[ChainId["Pirl"] = 3125659152] = "Pirl";
    ChainId[ChainId["OneLedgerTestnetFrankenstein"] = 4216137055] = "OneLedgerTestnetFrankenstein";
    ChainId[ChainId["PalmTestnet"] = 11297108099] = "PalmTestnet";
    ChainId[ChainId["Palm"] = 11297108109] = "Palm";
    ChainId[ChainId["NtityMainnet"] = 197710212030] = "NtityMainnet";
    ChainId[ChainId["HaradevTestnet"] = 197710212031] = "HaradevTestnet";
    ChainId[ChainId["MolereumNetwork"] = 6022140761023] = "MolereumNetwork";
    ChainId[ChainId["GodwokenTestnet(V1)"] = 868455272153094] = "GodwokenTestnet(V1)";
})(ChainId = exports.ChainId || (exports.ChainId = {}));
